### PR TITLE
Cleanup "Connect to External Services"

### PR DESCRIPTION
**Pull request by paolaperaza**
I couldn't help myself! I was reading through this doc, which I know we've touched a bunch recently, and noticed a few things I wanted to clean up:

- Added context for AWS Transit Gateways (what are they, what are its benefits) in line with other sections
- Add steps for Workload Identity, also to be consistent with steps in other sections
- Add the two new connection methods to the top of the doc (AWS Transit Gateway
- AWS IAM roles), they were missing
- Moved the workload identity section to the bottom such that the AWS only sections were next to each other
- Language cleanup elsewhere

Let me know what you all think. Sorry for putting you through another round of this, @dnarain-astro !

### CODE REVIEW

File: astro/create-cluster.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ To create an Astro cluster, you need to have:
 
 To create a new Astro cluster on AWS for your Organization, you must first reach out to your Astronomer representative. For each new cluster that you want to provision, provide our team with the following information:
```
Comment (kwcanuck) [inline astro/create-cluster.md:37]:
```suggestion
To create a new Astro cluster on AWS for your Organization, submit a [support request](astro-support.md). In your support request, provide the following information for every new cluster that you want to provision:
```

File: astro/create-cluster.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ To create an Astro cluster, you need to have:
...
+- Your AWS Account ID
+- Your preferred Astro cluster name
+- The AWS region that you want to host your cluster in
+- Your preferred node instance type
+- Your preferred max node count
 
-If not specified, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1` by default. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
+If not specified, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
 
 From there, your Astronomer representative will provide you with a unique `External ID` for each new cluster. Make note of this value for the next step.
```
Comment (kwcanuck) [inline astro/create-cluster.md:47]:
```suggestion
After your clusters are created, Astronomer support provides you with a unique `External ID` for every new cluster. You'll need these values to complete the next step.
```

File: astro/connect-external-services.md [hunk 3]
```diff
@@ -147,5 +122,44 @@ To grant an Astro cluster access to a service that is running in an AWS account
...
+2. Use your Deployment's namespace and the name of your Google Cloud project for Astro to identify the Kubernetes service account for your Deployment.
+    
+    Kubernetes service accounts for Astro Deployments are formatted as follows:
 
-8. Create an Airflow connection to AWS for each Deployment that requires the resources you connected. See [Managing connections to Apache Airflow](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html).
+    ```text
+    astro-<deployment-namespace>@<gcp-project-name>.iam.gserviceaccount.com
+    ```
+    
+    For a Google Cloud project called `astronomer-prod` and a Deployment namespace defined as `nuclear-science-2730`, for example, the service account for the Deployment would be:
```
Comment (kwcanuck) [inline astro/connect-external-services.md:144]:
```suggestion
    For a Google Cloud project named `astronomer-prod` and a Deployment namespace defined as `nuclear-science-2730`, for example, the service account for the Deployment would be:
```

File: astro/connect-external-services.md [hunk 4]
```diff
@@ -147,5 +122,44 @@ To grant an Astro cluster access to a service that is running in an AWS account
...
+Complete these steps for each Astro cluster that requires access to external data services on AWS.
+
+## Workload Identity (_GCP only_)
+
+[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity) is recommended by Google as the best way for data pipelines running on Google Cloud Platform (GCP) to access Google Cloud services in a secure and manageable way. All Astro clusters on GCP have Workload Identity enabled by default. Each Astro Deployment is associated with a Kubernetes service account that's created by Astronomer and is bound to an identity from your Google Cloud project's fixed workload identity pool.
+
+To grant a Deployment on Astro access to external data services on GCP, such as BigQuery:
+
+1. In the Cloud UI, go to your Deployment and copy paste the value under **Namespace**.
+2. Use your Deployment's namespace and the name of your Google Cloud project for Astro to identify the Kubernetes service account for your Deployment.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:136]:
```suggestion
2. Use the Deployment namespace value and the name of your GCP project to identify the Kubernetes service account for your Deployment.
```

File: astro/connect-external-services.md [hunk 5]
```diff
@@ -24,110 +26,83 @@ These endpoints can be configured by:
...
 
 1. In the Cloud UI, click **Clusters** and then copy the value displayed in the **Cluster ID** column for the Astro cluster that needs access to AWS service resources.
-
 2. Create an IAM role in the AWS account that contains your AWS service. See [Creating a role to delegate permissions to an AWS service](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html).
 
 3. In the AWS Management Console, go to the Identity and Access Management (IAM) dashboard.
-
-4. Click **Roles** and in the **Role name** column select the role you created in step 2.
-
+4. Click **Roles**. In the **Role name** column, select the role you created in step 2.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:104]:
```suggestion
4. Click **Roles** and in the **Role name** column, select the role you created in step 2.
```
This is the correct format for this type of instruction. If you add punctuation, create a new task step

File: astro/connect-external-services.md [hunk 6]
```diff
@@ -147,5 +122,44 @@ To grant an Astro cluster access to a service that is running in an AWS account
 }
 ```
 7. Click **Update policy**.
+8. In the Airflow UI or as an environment variable on Astro, create an Airflow connection to AWS for each Deployment that requires the resources you connected. See [Managing connections to Apache Airflow](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html).
+
+Complete these steps for each Astro cluster that requires access to external data services on AWS.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:127]:
```suggestion
Complete these steps for each Astro cluster that requires access to external data services on AWS.
```

This should be included in the introductory statement, or added a s a procedure step. For example:

9. Optional. Repeat steps 1 to 8 for every Astro cluster that requires access to external data services on AWS.

File: astro/connect-external-services.md [hunk 7]
```diff
@@ -147,5 +122,44 @@ To grant an Astro cluster access to a service that is running in an AWS account
...
+
+## Workload Identity (_GCP only_)
+
+[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity) is recommended by Google as the best way for data pipelines running on Google Cloud Platform (GCP) to access Google Cloud services in a secure and manageable way. All Astro clusters on GCP have Workload Identity enabled by default. Each Astro Deployment is associated with a Kubernetes service account that's created by Astronomer and is bound to an identity from your Google Cloud project's fixed workload identity pool.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:131]: Define acronyms such as Google Cloud Platform (GCP) early in the document and then use only the acronym afterward.

File: astro/connect-external-services.md [hunk 8]
```diff
@@ -147,5 +122,44 @@ To grant an Astro cluster access to a service that is running in an AWS account
...
+    For a Google Cloud project called `astronomer-prod` and a Deployment namespace defined as `nuclear-science-2730`, for example, the service account for the Deployment would be:
+
+    ```text
+    astro-nuclear-science-2730@astronomer-prod.iam.gserviceaccount.com
+    ```
+3. Go to the Google Cloud project in which your external data service is hosted.
+4. Add the Kubernetes service account for your Astro Deployment to the principal of that Google Cloud project. See [GCP documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to).
+5. Bind the service account to a role that has access to your external data service.
+
+Complete these steps for every Astro Deployment that requires access to external data services on GCP.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:153]: See previous edit recommendation. Add this information to the task introductory statement, or add it as an optional procedure step.

File: astro/connect-external-services.md [hunk 9]
```diff
@@ -147,5 +122,44 @@ To grant an Astro cluster access to a service that is running in an AWS account
...
+
+:::info
+
+GCP has a 30 character limit for service account names. For Deployment namespaces which are longer than 24 characters, use only the first 24 characters when determining your service account name.
+
+For example, if your GCP project was called `astronomer-prod` and your Deployment namespace was `nuclear-scintillation-2730`, your service account would be:
```
Comment (kwcanuck) [inline astro/connect-external-services.md:159]:
```suggestion
For example, if your GCP project is named `astronomer-prod` and your Deployment namespace is `nuclear-scintillation-2730`, your service account is:
```

File: astro/connect-external-services.md [hunk 10]
```diff
@@ -24,110 +26,83 @@ These endpoints can be configured by:
...
-- Go to the Google Cloud project in which your external data service is hosted
-- Add the Kubernetes service account for your Astro Deployment to the principal of that Google Cloud project
-- Bind the service account to a role that has access to your external data service
-
-Kubernetes service accounts for Astro Deployments are formatted as follows:
-
-```text
-astro-<deployment-namespace>@<gcp-project-name>.iam.gserviceaccount.com
-```
+6. Contact [Astronomer support](https://support.astronomer.io) to confirm that you have created the static route. From here, Astronomer updates the routing table of the Astro VPC to send traffic from your CIDR block through the transit gateway.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:92]:
```suggestion
6. Contact [Astronomer support](https://support.astronomer.io) to confirm that you have created the static route. Astronomer support will update the Astro VPC routing table to send traffic from your CIDR block through the transit gateway.
```

File: astro/connect-external-services.md [hunk 11]
```diff
@@ -24,110 +26,83 @@ These endpoints can be configured by:
...
 
 - An Astro cluster
-- A transit gateway in the same region as your Astro cluster
+- An existing transit gateway in the same region as your Astro cluster
 - Permission to share resources using AWS Resource Access Manager (RAM)
 
 :::info
 
-If your transit gateway is in a different region than your Astro cluster, contact [Astronomer support](https://support.astronomer.io). Astronomer can create a new transit gateway and set up cross-region peering with your existing transit gateway. With this configuration your organization can incur AWS charges for a new transit gateway hosted in the Astro account, as well as inter-region transfer costs.
+If your transit gateway is in a different region than your Astro cluster, contact [Astronomer support](https://support.astronomer.io). Astronomer can create a new transit gateway in your AWS account for Astro and set up a cross-region peering connection with your existing transit gateway.
```
Comment (kwcanuck) [inline astro/connect-external-services.md:79]:
```suggestion
If your transit gateway is in a different region than your Astro cluster, contact [Astronomer support](https://support.astronomer.io). Astronomer support can create a new transit gateway in your AWS account for Astro and set up a cross-region peering connection with your existing transit gateway.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/connect-external-services.md (hunk 6 above)
Comment (paolaperaza) [reply astro/connect-external-services.md]: Good call! Done

**Pull request merged**